### DISCUSSION TITLE
Videos Glitch and Malfunction When Downloaded

### DISCUSSION THREAD

**Discussion by Unchained1966** [aclap-dev/video-downloadhelper#52] [Help and Q&A] [answered]
### Your question

Hi, when I download the videos that I want to download from the specific website, the downloading goes fine. But when I want to watch it I experience 2 different problems:
1. Some videos playback very glitchy. The audio is fine, but the images seem to progress in small freezes or skips. As if every half second it freezes for half a second and continues.
I experience this in Apple Preview and in Apple Quicktime, in VLC media player these videos work fine.

2. Other videos are worse. The audio again is good, but the video fastforwards al the images in a couple seconds/minutes (45 min original clip is shown in 1 or 2 minutes) then the screen goes completely black. The audio keeps playing for the full duration of the 45 minute video, but the images are played super fast as explained and then just a black screen for the duration of the video.
I experience this in Apple Preview and in Quicktime, in VLC media player these videos play the audio correctly but the images just show a freeze of the first frame.

So these are both some problems I experience. The first one can be easily mitigated by watching in VLC, which is fine by me. But the second problem doesn't give me an easy solution. For both problems I had expected that this would not be the case when using Video Downloadhelper. 

Does someone know how I can fix this?

I'm running apple M2 Macbook pro, MacOS Ventura 13.6.2.

Thanks in advance for the help

**Comment by paulrouget** [comment]
> Some videos playback very glitchy

I need the URL of these videos.

> Other videos are worse.

I need the URL of these videos too.

> Addon & CoApp version and details
> No response

We can't help you without knowing which version of the addon and the coapp you are using. Please share the details about your setup. Read this: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Reply by GrampaWildWilly** [reply]
Part of your problem description is a 100% match for something we figured out quite some time ago & posted about back on the old Google forum (https://groups.google.com/g/video-downloadhelper-q-and-a).  You need to go back there & search that forum using "too fast" as your search key.  Put the search key in " " to shrink the results list a bit.  We referred to the thread in question a lot so those references show up as search results.  You need to find the discussion that offers suggestions & workarounds for the "too fast" problem.  There's a bunch of ideas we put forward in that discussion.  Every web site seems to exhibit the problem in a different way & there's many variations on the theme of how to get around the problem.  The short version is you use ffmpeg instead of VDH to download the content.

Plus, as Paul said, you need to provide a proper problem report.  That includes the information that is requested when you create a discussion here, information you believed you didn't need to supply.  You need to supply it.  Also as Paul said, you need to provide URLs of the web pages where the videos are hosted.  If he's going to solve shortcomings in VDH, he's going to need test cases.  Don't be coy about posting URLs.  Post them.

**Reply by Unchained1966** [reply]
@paulrouget Thanks for your quick response. my apologies on not sending the version information. This is it: 
<img width="532" alt="Screenshot 2024-02-09 at 09 09 23" src="https://github.com/aclap-dev/video-downloadhelper/assets/159472812/ea1969c0-23b6-4cd7-a4b7-6512a5a0ba3c">

The links are in a private login environment, I don't know if that's of any good use (first time on this forum). But here it is, hope it helps:

1. Glitchy video: https://studioorbit.org/making-funky-electro-house-like-b-love-kolter-sweely/

2. Worse video: https://studioorbit.org/how-to-make-peak-time-speed-garage/

Is this the information you need or do I need to add something or put something in a different way?

**Reply by Unchained1966** [reply]
@GrampaWildWilly Thanks for your response and help! My apologies for not providing the correct information. First time on forum, thanks for pointing this out to me so I can refrain from omitting any needed information in future threads.

I've been searching the old google forum, but I find it difficult to find the solution that I need. I did come across another user who recently experienced the same issue and I believe from what I'm seeing that @paulrouget is trying to solve the problem. Will monitor that and this thread and keep searching on google forum. Hopefully I can find a solution soon.

Thanks!

**Reply by GrampaWildWilly** [reply]
The exact title of the thread in the Google forum is "Too fast video - possible solutions or workarounds."  Yes, Paul is working on an answer.  Unfortunately, the URLs you posted are on a pay site, as you said.  And the prices don't look cheap.  Unfortunately, that's going to make it impossible for anybody here to use your examples as test cases.  See this for the reason: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#start-a-new-qa-post

You might try getting onto the latest versions of things.  The latest beta of VDH is here: https://github.com/aclap-dev/video-downloadhelper/wiki/Beta-versions.  Currently, that's version 8.2.0.19a1 (on Firefox, might be different for Chrome) but things have been changing fast around here so you might go there & get something newer.  You should disable automatic updates to VDH in any case.  As a few users have found out, to their dismay, the browser might update VDH while you're in the middle of doing a download.  That kills the download.  The browser apparently doesn't check whether the add-on is in use.  The latest version of the CoApp is here: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-Installation.  At the moment, that's version 2.0.17.

**Reply by Unchained1966** [reply, marked as answer]
@GrampaWildWilly thanks! I will check it out. The Beta version in Chrome works! Thanks to you and @paulrouget for helping me out.

**Reply by masterofobzene** [reply]
No problems if I use the download link of any video and paste it into aria2c. This is a problem with this extension.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I am just another user.  Also, please read #2214.

This is a problem with this extension as it existed when this thread was new.  Look at the dates on the posts.  This problem has long since been solved in the current version of VDH & in several versions in the interim between then & now.

**Reply by masterofobzene** [reply]
> This problem has long since been solved in the current version of VDH

That's why I'm here, I know this is old, but the problem is there again or it was never fixed.
If I use the same links copied from vdh and pass it to aria2c, the video gets downloaded without any corruption. Also, I'm using the firefox version.


PS: don't worry, not going to report spam or anything. Maybe it was a bot who got you banned that time; now its much more common.

**Reply by GrampaWildWilly** [reply]
Unfortunately, the videos mentioned by the original poster are behind a site registration.  That makes them inaccessible to me so I can't test them.  However, the original poster said a year & a half ago that the problem was solved on the Chrome beta available then.  So it was solved a year & a half ago.  Are you a member of that web site?  Are you trying to do the downloads with a current version of VDH?  Or are you just here to try to hype up aria2c?

**Reply by masterofobzene** [reply]
I see why you got banned.

No, I'm trying to get videos from a random tube site using the latest version. 
Are you part of the team or are you spamming all threads to hype this extension?
Aria2c is downloading them ok, so sorry to spoil your fanatism but  if another downloader does the job with the exact same link this extension is using, _something is actually wrong with this extension_

**Reply by RobertL-CH** [reply]
@masterofobzene – Ad hominem attacks on other users are not acceptable. Rudeness is contrary to the Github guidelines. These things reduce the effectiveness of the forum in providing a place where users help each other. Kindly desist.

**Reply by GrampaWildWilly** [reply]
"Random tube site."  Let's have at least one URL from that site & I'll see if I get the same weirdness in my download.

**Reply by masterofobzene** [reply]
You are just one guy with many accounts attacking users.

**Reply by GrampaWildWilly** [reply]
Ummmm......  Can we get back on topic here?  Post at least one URL of a web page on your "random tube site" so I can try to replicate your results.  Make it something you have downloaded with VDH that you played back & it exhibited the weird artifacts.  Let's see if I get the same weird artifacts.  We need to get the same content for it to be a valid test.  So let's have that information & we can get back to the topic at hand.
